hotham/src/asset_importer/mod.rs

```diff
@@ -134,7 +130,7 @@ fn load_models_from_gltf_data(import_context: &mut ImportContext) -> Result<()>
     // We need *some* entity to stash the AnimationController onto.
     // For now, just use the first root entity.
     let mut animation_controller_entity = None;
-    for node in document.scenes().next().unwrap().nodes() {
+    for node in document.nodes() {
```

Comment:
Ah! With this one you definitely need to use `scene.nodes()`. The reason is because `document.nodes()` doesn't have any hierarchy information, so the nodes won't get loaded in correctly. It should be the *one* exception to this though.

Reply:
ah yep that makes sense! well I've reversed it back and tested it with crab_saber and simple_scene and it seems to work fine